### ISSUE TITLE
Lua error when unloading TMR into a cow barn with a mixer wagon

### ISSUE THREAD

**Issue by Timmeey86** [VertexDezign/TargetFillLevel#4]
Hi there,

looks like version 2.0.1.0 causes the following error when trying to unload TMR into a cow barn from a mixer wagon:

```txt
Error: Running LUA method 'update'.
<moddir>/FS22_TargetFillLevel/scripts/main.lua:213: attempt to call method 'getFillLevel' (a nil value)
```

I guess cow barns don't have a fill level since they've got several sliders instead.

Steps to reproduce:

1. Create a new save game on elmcreek with just TFL and EasyDev
2. Cheat some cash, buy a field next to the shop and place a cow barn
3. Buy/Lease a tractor and a mixer wagon
4. Fill the mixer wagon with TMR
5. Drive into the cowbarn and try overloading the TMR

Regards,
Farmsim Tim

**Comment by whitecanealpha** [comment]
I got this same error trying to dump silage out of a wheel-loader bucket into a silage pile (cow barn) and trying to tip wheat out of a wheelbarrow into a chicken pen feed trough, so it's dead easy to reproduce.

**Comment by schliesser** [comment]
Heyho @Timmeey86 and @whitecanealpha 

thank you for reporting this issue. I've implemented a quick fix to prevent the errors. Please try the latest development version here: https://github.com/VertexDezign/TargetFillLevel/releases/download/latest/FS22_TargetFillLevel.zip

There is currently no TFL display when unloading to animal husbandaries.

**Comment by Timmeey86** [comment]
Works perfectly now 👍 

I don't think a TFL display for animal husbandries is needed / easily possible either. You can access the animal husbandry overview menu without having to leave your tractor so it's fine like that in my opinion

**Comment by whitecanealpha** [comment]
Agree with Timmeey86

**Comment by schliesser** [comment]
Hotfix is also released on ModHub with version 2.0.1.1 : https://www.farming-simulator.com/mod.php?mod_id=231717

Thank you all and have a nice farming day 🚜 🐄
